evap/staff/views.py

```diff
@@ -1051,6 +1051,43 @@ def course_copy(request, course_id):
     )
 
 
+@require_POST
+@manager_required
+@transaction.atomic
+def create_exam_evaluation(request):
+    evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
+    exam_date = request.POST.get("exam_date")
+    exam_date = datetime.combine(datetime.strptime(exam_date, "%Y-%m-%d"), datetime.min.time())
+
+    if evaluation.is_single_result:
+        raise SuspiciousOperation("Creating an exam evaluation for a single result evaluation is not allowed")
+
+    if evaluation.has_exam:
+        raise SuspiciousOperation("An exam evaluation already exists for this course")
+
+    evaluation_end_date = exam_date - timedelta(days=1)
+    if evaluation.vote_start_datetime > evaluation_end_date:
+        messages.error(
+            request, _("The exam date is before the start date of the main evaluation. No exam evaluation created.")
+        )
+        return HttpResponse()
+
+    evaluation.weight = 9
+    evaluation.vote_end_date = evaluation_end_date
+    evaluation.save()
+
+    exam_evaluation = Evaluation(
+        course=evaluation.course, name_de="Klausur", name_en="Exam", weight=1, is_rewarded=False
+    )
+    exam_evaluation.make_exam_evaluation(
+        exam_date=exam_date,
+        participants=evaluation.participants.all(),
+        eval_contributions=evaluation.contributions.exclude(contributor=None),
+    )
```

Comment:
`make` in our codebase is, coming from model_bakery, connotated with "create some new instance".

Have you considered putting all this stuff into a static method `Evaluation::make_exam_evaluation()` that does these steps?